Apps/Admin/Client/Store/Dashboard/DashboardActions.cs

```diff
@@ -87,64 +97,110 @@ public LoadDependentsAction(int timeOffset)
     }
 
     /// <summary>
-    /// The action representing the initiation of recurring users load action.
+    /// The action representing a successful logged in user action.
     /// </summary>
-    public class LoadRecurringUsersAction
+    public class LoggedInSuccessAction : BaseSuccessAction<IDictionary<DateTime, int>>
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="LoadRecurringUsersAction"/> class.
+        /// Initializes a new instance of the <see cref="LoggedInSuccessAction"/> class.
         /// </summary>
-        /// <param name="days">The number of unique days for evaluating a user.</param>
-        /// <param name="startPeriod">The period start over which to evaluate the user.</param>
-        /// <param name="endPeriod">The period end over which to evaluate the user.</param>
-        /// <param name="timeOffset">The offset from the client browser to UTC.</param>
-        public LoadRecurringUsersAction(int days, string startPeriod, string endPeriod, int timeOffset)
+        /// <param name="data">user data.</param>
+        public LoggedInSuccessAction(IDictionary<DateTime, int> data)
+            : base(data)
         {
-            this.Days = days;
-            this.StartPeriod = startPeriod;
-            this.EndPeriod = endPeriod;
-            this.TimeOffset = timeOffset;
         }
+    }
 
+    /// <summary>
+    /// The action representing a failed logged in user action.
+    /// </summary>
+    public class LoggedInUserFailAction : BaseFailAction
```

Comment:
Typo: In this action, `LoggedInUser` should be `LoggedInUsers` to match the related actions.